src/chapters/arrays/multi-dimensional-arrays.rst

```diff
@@ -0,0 +1,107 @@
+Multi-Dimensional Arrays
+========================
+
+Earlier we learned that arrays can store any type of value. If that is true, can we store arrays inside
```

Comment:
I think this paragraph can be shortened. "In addition to storing numbers, strings, and boolean values, arrays can also store arrays."

Reply:
I removed the second sentence about "multi-dimonsional universe". I want to keep the first sentence because I really like the flow of asking the user a question and then defining what a multidimensional array is.